helm/cluster-cloud-director/values.schema.json

```diff
@@ -96,6 +96,78 @@
+                            "properties": {
+                                "endpoint": {
+                                    "type": "string",
+                                    "title": "Endpoint",
+                                    "description": "Endpoint for the container registry."
+                                },
+                                "credentials": {
+                                    "type": "object",
+                                    "title": "Credentials",
+                                    "description": "Credentials for the endpoint.",
+                                    "properties": {
+                                        "username": {
+                                            "type": "string",
+                                            "title": "Username",
+                                            "description": "Username is the username to login the registry."
+                                        },
+                                        "password": {
+                                            "type": "string",
+                                            "title": "Password",
+                                            "description": "Password is the password to login the registry."
+                                        },
+                                        "auth": {
+                                            "type": "string",
+                                            "title": "Auth",
+                                            "description": " Auth is a base64 encoded string from the concatenation of the username, a colon, and the password."
+                                        },
+                                        "identitytoken": {
+                                            "type": "string",
+                                            "title": "Identity Token",
+                                            "description": "Identity Token is used to authenticate the user and get an access token for the registry."
+                                        }
+                                    }
+                                }
+                            },
+                            "required": [
+                                "endpoint"
+                            ]
+                        }
+                    },
+                    "examples": [
+                        {
+                            "docker.io": [
+                                { 
+                                    "endpoint": "registry-1.docker.io",
+                                    "credentials":{
+                                        "username": "my_user_name",
+                                        "password": "my_password"
+                                    }
+                                },
+                                { "endpoint": "my-mirror-registry.mydomain.io"}
+                            ]
+                        }
```

Comment:
Not sure we'll have a use for such a complex examples (in contrast to string or number field examples). In the UI we will not have any way to display it.

Reply:
If so, I am removing this.